paymentapi/src/main/java/fi/hel/verkkokauppa/payment/api/PaytrailPaymentController.java

```diff
@@ -51,14 +51,15 @@ public ResponseEntity<Payment> createPaymentFromOrder(@RequestBody GetPaymentReq
 
     @GetMapping("/payment/paytrail/check-return-url")
     public ResponseEntity<PaymentReturnDto> checkReturnUrl(
+            @RequestParam(value = "merchantId") String merchantId,
```

Comment:
This is required in request params from Exp-api  inorder to fetch merchant configurations and the proper secret key for normal merchant flow.